### DISCUSSION TITLE
Clain LP -  Wish Well

### DISCUSSION THREAD

**Discussion by Chucrutty** [sunflower-land/sunflower-land#2488] [Polls]
I think today's WW model leaves a lot to be desired and also discourages people from providing liquidity to the project, this WW thing having to collect every X time or else stopping the % receipt count doesn't make any sense.

what other models could be proposed?

**Comment by kegw7689** [comment]
i guess it has something to do with the fact that it is a game and not financial product. You make a wish that takes 3 days to give you something is the game loop

**Reply by Chucrutty** [reply]
Becose of that they can't do a good mechanic????

**Comment by ispankzombiez** [comment]
perhaps the wishing well can be moved to the main island? allow the player the option to see it on their farm.

or, perhaps there can be a notification added to let the player know when they can make another wish?

**Comment by adamhannigan** [comment]
Hey @Chucrutty - just an update on this one. The team has been experimenting with Merkl lately. This should help solve the problems that we currently face.

https://merkl.angle.money/
